packages/sqrl-cli-functions/package.json

```diff
@@ -20,30 +20,19 @@
     "sqrl"
   ],
   "files": [
-    "lib"
+    "lib/**/*.js",
+    "lib/**/*.d.ts"
   ],
   "main": "lib/index.js",
   "typings": "lib/index.d.ts",
   "scripts": {
     "clean": "rimraf lib && rimraf coverage",
-    "format": "prettier --write \"{src,__tests__}/**/*.ts\"",
-    "lint": "tslint --force --format verbose \"src/**/*.ts\"",
-    "build": "tsc --pretty",
     "test": "jest",
     "test:integration": "jest --config=jest.integration.config.js",
     "coverage": "jest --coverage",
-    "watch": "npm run build -- --watch",
```

Comment:
Seems odd to kill some of these but not others?

It can be useful to `npm run watch` from within a specific package

Reply:
ooh yeah anything build-related at the individual package level should be removed. I’ll do another sweep.

`tsc --build` manages all things build-related now, so we don’t need to build individual projects. It builds projects in dependency order, starting at the edges and then working in towards the dependent packages.